manifests/server/config.pp

```diff
@@ -147,6 +147,17 @@
                   Exec['puppet_server_config-create_ssl_dir'],
                   ],
     }
+  } elsif $::puppet::server::ca_crl_sync {
+    # If not a ca AND sync the crl from the ca master
+    if $::server_facts['servername'] != undef {
```

Comment:
$::puppetmaster is another one of those fun ones... ON non foreman installs this topscope custom var is nonexistant... making the defined function flood the log, as we are currently discussing in #467 however $::servername would work and is supported at least back to puppet 3.5 what are your thoughts about this?